src/main/java/frc/team2412/robot/commands/intake/FeederOutCommand.java

```diff
@@ -15,4 +15,9 @@ public FeederOutCommand(IntakeSubsystem intakeSubsystem) {
 	public void initialize() {
 		intakeSubsystem.feederOut();
 	}
+
+	@Override
+	public boolean isFinished() {
+		return false;
+	}
```

Comment:
The feeder out, intake out & intake stop commands should all continue to return 'true' from isFinished (the defafult behavior, so can delete the override `isFinished` methods)